cleanlab_studio/errors.py

```diff
@@ -24,3 +24,11 @@ def __init__(self) -> None:
 class AuthError(APIError):
     def __init__(self) -> None:
         super().__init__("invalid API key")
+
+
+class InternalError(RuntimeError):
```

Comment:
I think it's more common to inherit from `Exception`; I think `RuntimeError` is Python's "not any of the other built-in errors, like `ValueError` or `IndexError` or ...", but I usually see custom exceptions inheriting from either one of the specific Python errors (like `ValueError`) or directly from `Exception`.